e2e/main.go

```diff
@@ -48,6 +54,19 @@ func main() {
 		Dir:  filepath.Join(*pagesDir, "dynamic"),
 	})
 
+	var filterR *regexp.Regexp
+
+	if *filter != "" {
+		r, err := regexp.Compile(*filter)
+
+		if err != nil {
```

Comment:
I think would be better to use ```regexp.MustCompile(*filter)``` instead.

Reply:
That would panic the app without a proper message.

Reply:
It will.

Reply:
It will.